### PR TITLE
Instructor: edit visibility/path: warn if user click on first level options #6095

### LINKED ISSUE #6095: Instructor: edit visibility/path: warn if user click on first level options
Some instructors click on these and complain nothing happens
image

### PR DESCRIPTION
**Pull request by VamsiSangam**
Fixes #6095 

**Outline of Solution**

As discussed in issue, 2 solutions were suggested -

- Make the menu not disappear on clicking a first level option.
- Display a modal saying instructor is supposed to click on a second level option and some descriptive info about it.

I tried a lot to keep the menu from not disappearing when clicked on a first level option, but it simply can't be done. That particular click event on the first level options comes from `bootstrap.min.js`, so I don't think it can be done.

Currently, I am displaying a tooltip over the submenu indicating the instructor must select a submenu option.

hohnzf

Clicking on a first level option still causes the menu to disappear.

### CODE REVIEW

File: src/main/webapp/jsp/instructorFeedbackEdit.jsp [hunk 1]
```diff
@@ -33,6 +33,13 @@
     <script type="text/javascript" src="/js/instructorFeedbackEdit/questionContrib.js"></script>
     <script type="text/javascript" src="/js/instructorFeedbackEdit/questionRubric.js"></script>
     <script type="text/javascript" src="/js/instructorFeedbackEdit/questionRank.js"></script>
+    <script>
+        $(function () {
+            $(".first-level-dropdown-option").hover(function() {
+                $(this).siblings(".dropdown-menu").tooltip("show");
+            });
+        });
+    </script>
```
Comment (VamsiSangam) [inline src/main/webapp/jsp/instructorFeedbackEdit.jsp]: Ok will do :+1:

**Comment by VamsiSangam** [comment]
## Outline of changes

- Moved code to JS code to `instructorFeedbackEdit.es6`.
- UI remains same with a minor enhancement that the tooltip now, does not blink when user moves mouse from the first level option to the sub-menu option.

Ready for review.

File: src/test/resources/pages/studentFeedbackSubmitPageSuccessRank.html [hunk 2]
```diff
@@ -5,6 +5,12 @@
     Submit Feedback
   </h1>
   <br>
+  <div id="student-motd-wrapper">
+    <input id="motd-url" type="hidden" value="${studentmotd.url}">
+    <script defer="" src="/js/studentMotd.js" type="text/javascript">
+    </script>
+    ${studentmotd.container}
+  </div>
```
Comment (thenaesh) [inline src/test/resources/pages/studentFeedbackSubmitPageSuccessRank.html]: This doesn't look related to the changeset. Can you check which test it came from?

Comment (VamsiSangam) [inline src/test/resources/pages/studentFeedbackSubmitPageSuccessRank.html]: It comes from `teammates.test.cases.browsertests.FeedbackRankQuestionUITest` > `testStudentSubmitAndResultsPages()`. Yes it does look like undesired code. Nice catch! :+1: But I'm not sure how to deal with it. All my tests are passing without godmode now. :sweat_smile:

Comment (thenaesh) [inline src/test/resources/pages/studentFeedbackSubmitPageSuccessRank.html]: We are currently investigating this now. This has popped up quite a few times recently and we have no idea why.

Comment (wkurniawan07) [inline src/test/resources/pages/studentFeedbackSubmitPageSuccessRank.html]: This block is expected to be present. It was erroneously removed in the first commit of #7206 for some reason, good thing that it's restored here.

Comment (VamsiSangam) [inline src/test/resources/pages/studentFeedbackSubmitPageSuccessRank.html]: My bad :disappointed: I will be more careful using the god mode next time.

Comment (thenaesh) [inline src/test/resources/pages/studentFeedbackSubmitPageSuccessRank.html]: @VamsiSangam it's not your fault, don't worry. :)
